pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
I'm agnostic on how you guys choose to solve this problem, but I would like @rifung to be looking at the k8s reaction chain fakes.